_posts/en/newsletters/2024-06-07-newsletter.md

```diff
@@ -165,17 +165,45 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
 [Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
 repo], and [BINANAs][binana repo]._
 
-FIXME:Gustavojfe
-
-- [Core Lightning #7252][] lightningd: don't ignore fee limits on mutual close.
-
-- [LDK #2931][] log-outbound-channels
-
-- [Rust Bitcoin #2644][] Add HMAC Extract-and-Expand Key Derivation Function
-
-- [BIPs #1541][] BIP431: Opt In Topologically Restricted Until Confirmation Transactions For More Robust Fee-bumping
-
-- [BIPs #1556][] BIP 337: Compressed Transactions
+- [Core Lightning #7252][] changes the behavior of `lightningd` to ignore the
+  `ignore_fee_limits` setting during a cooperative channel closure. This fixes
+  an issue where a Core Lightning (CLN) channel opener node overpays fees when
+  the counterparty is a LDK node. In this scenario, when the LDK non-opener node
+  (Alice) initiates a cooperative channel closure and begins fee negotiation,
+  the CLN opener node (Bob) responds that the fee can be anything between
+  `min_sats` and `max_channel_size` due to the `ignore_fee_limits` setting. Bob
+  picks the upper bound (contrary to the BOLTs specification), and Alice
+  accepts, resulting in Alice broadcasting a transaction with considerably
+  overpaid fees.
+
+- [LDK #2931][] enhances the logging during path finding to include additional
```

Comment:
```suggestion
- [LDK #2931][] enhances the logging during pathfinding to include additional
```
or maybe "path-finding"?

Reply:
good catch. 'pathfinding' is the right one. https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md